ingestion/src/metadata/ingestion/source/database/dbt_source.py

```diff
@@ -199,3 +213,179 @@ def create_dbt_lineage(self) -> Iterable[AddLineageRequest]:
                     logger.warning(
                         f"Failed to parse the node {upstream_node} to capture lineage: {exc}"
                     )
+
+    def create_dbt_tests_suite_definition(self):
+        """
+        After everything has been processed, add the tests suite and test definitions
+        """
+        try:
+            if (
+                self.source_config.dbtConfigSource
+                and self.dbt_manifest
+                and self.dbt_catalog
+            ):
+                logger.info("Processing DBT Tests Suites and Test Definitions")
+                test_suite = CreateTestSuiteRequest(
+                    name="DBT_TESTS",
+                    description="Test Suite for DBT Tests",
+                )
+                yield test_suite
+                for key, dbt_test in self.dbt_tests.items():
+                    test_definition = CreateTestDefinitionRequest(
+                        name=dbt_test["name"],
+                        description=dbt_test["description"],
+                        entityType=EntityType.COLUMN,
```

Comment:
@OnkarVO7 how are you passing table tests?

Reply:
I did'nt see any table tests here..Do you have some docs around it?

Reply:
@OnkarVO7 some of [these](https://github.com/dbt-labs/dbt-utils#equal_rowcount-source) tests are table level tests

Reply:
Thanks @TeddyCr I'll take a look